### ISSUE TITLE
LocalNavigator creates new Navigator object with new key

### ISSUE THREAD

**Issue by ceyuboglu** [adrielcafe/voyager#312]
LocalNavigator.currentOrThrow or LocalNavigator.current creates new object with new key. This should be same navigator everywhere if i wrapped my app once with Navigator() {} , right ?

**Comment by DevSrSouza** [comment]
Yes! It should, same navigator. Your LocalComposition does not have "default Navigator", it throws a Exception if it is not provided, what could be happening is that your is recomposing the Navigator it self (the `Navigator() {}` declaration), so in this case, it could be it is being recreating a new navigator all the time.

**Comment by DevSrSouza** [comment]
Can you share reproducible code that we can test inside voyager samples?
